README.md

```diff
@@ -16,76 +17,90 @@ pip install --upgrade vellum-ai
 ## Async Client
+This SDK has an async version. Here's how to use it:
+
+
 
 ```python
+import asyncio
+
 import vellum
 from vellum.client import AsyncVellum
 
-raven = AsyncVellum(api_key="YOUR_API_KEY")
-
-async def generate() -> str:
-  result = client.generate(
-    deployment_name="my-deployment",
-    requests=[
-        vellum.GenerateRequest(
-            input_values={"question": "Can I get a refund?"})]
+client = AsyncVellum(api_key="YOUR_API_KEY")
+
+async def execute() -> str:
+    result = await client.execute_prompt(
+        prompt_deployment_name="<example-deployment-name>>",
+        release_tag="LATEST",
+        inputs=[
+            vellum.PromptDeploymentInputRequest_String(
+                name="input_a",
+                value="Hello, world!",
+            )
+        ],
     )
-  
-  return result.text
-```
 
-## Uploading documents
+    if result.state == "REJECTED":
+        raise Exception(result.error.message)
+    
+    return result.outputs[0].value
 
-Documents can be uploaded to Vellum via either the UI or this API. Once uploaded and indexed, Vellum's Search allows you to perform semantic searches against them.
-
-```python
-from vellum.client import Vellum
-
-client = Vellum(api_key="YOUR_API_KEY")
-
-with open("/path/to/your/file.txt", "rb") as file:
-    result = client.documents.upload(
-        # File to upload
-        contents=file,
-        # Document label
-        label="Human-friendly label for your document",
-        # The names of indexes that you'd like this document to be added to.
-        add_to_index_names=["<your-index-name>"],
-        # Optionally include a unique ID from your system to this document later.
-        #   Useful if you want to perform updates later
-        external_id="<your-index-name>",
-        # Optionally include keywords to associate with the document that can be used in hybrid search
-        keywords=[],
-    )
-
-print(result)
+if __name__ == "__main__":
+    print(asyncio.run(execute()))
 ```
 
-## Beta status
-
-This SDK is in beta, and there may be breaking changes between versions without a major version update. Therefore, we recommend pinning the package version to a specific version in your pyproject.toml file. This way, you can install the same version each time without breaking changes unless you are intentionally looking for the latest version.
-
 ## Contributing
 
-While we value open-source contributions to this SDK, this library is generated programmatically. Additions made directly to this library would have to be moved over to our generation code, otherwise they would be overwritten upon the next generated release. Feel free to open a PR as a proof of concept, but know that we will not be able to merge it as-is. We suggest opening an issue first to discuss with us!
+While we value open-source contributions to this SDK, most of this library is generated programmatically.
+
+Please feel free to make contributions to any of the directories or files below:
```

Comment:
[nit] I would also point people to vellum-client-generator. That way issues and prs-that-communicate-pocs could be consolidated on one repo